x-pack/plugins/reporting/server/usage/reporting_usage_collector.test.ts

```diff
@@ -45,6 +42,9 @@ describe('license checks', () => {
   describe('with a basic license', () => {
```

Comment:
Let's add this here:
```
  let exportTypesRegistry: ExportTypesRegistry;
  beforeAll(async () => {
    const mockReporting = await createMockReportingCore(createMockConfigSchema());
    exportTypesRegistry = mockReporting.getExportTypesRegistry();
  });
```

so that we don't need to declare an exportTypesRegistry for each section

Reply:
I think I fixed this now in commit [e81a97a](https://github.com/elastic/kibana/pull/162276/commits/e81a97aaa6812623426386f816651c6ec0f96cbc) thanks!!